hw/dv/tools/dvsim/riviera.hjson

```diff
@@ -9,12 +9,12 @@
   // {tool_srcs_dir} before running the simulation.
   tool_srcs:  ["{dv_root}/tools/riviera/*"]
 
-  build_opts: ["-sv",
-               "-timescale 1ns/1ps",
-               "-uvmver 1.2",
+  build_opts: ["-timescale 1ns/1ps",
+               "+incdir+\"{ALDEC_PATH}/vlib/uvm-1.2/src\"",
+               "\"{ALDEC_PATH}/vlib/uvm-1.2/src/uvm_pkg.sv\"",
```

Comment:
The general convention used across other tools is `{<SIMULATOR>_HOME}`.

Reply:
I changed this as you suggested